Automattic/sensei
# Add quiz grade tables based repository

Part of #5434

### Changes proposed in this Pull Request

* Add custom tables based repositories for quiz grades.
* Add aggregate repositories that use both types of repositories (comments based and tables based).

### Testing instructions

* Enable the tables based progress feature.
```php
add_filter( 'sensei_feature_flag_tables_based_progress', '__return_true' );
```
* Make a course, a lesson, and a quiz that has every question type.
* Submit the quiz in the front-end.
* Go to back-end Sensei LMS -> Grading and save grades
* Check `sensei_lms_quiz_grades` table to have all grades.
